Google.PowerShell/BigQuery/BqJob.cs

```diff
@@ -396,16 +441,59 @@ public Apis.Bigquery.v2.Data.Job DoQuery()
         }
 
         /// <summary>
-        /// Copy Job main processing function.
+        /// Copy Job main processing function.  
+        /// *This is written using Apis.BigQuery.v2 becuase Cloud.BigQuery did not 
+        /// support Copy opertations at the time of writing.
         /// </summary>
         public Apis.Bigquery.v2.Data.Job DoCopy()
         {
-            throw new NotImplementedException(
-                "Copy jobs are not implemented yet.  Use the *-BqTabledata cmdlets instead.");
+            if (ShouldProcess($"\n\nCopying table(s) to {Destination.TableId}\n\n"))
+            {
+                try
+                {
+                    var copyjob = new Apis.Bigquery.v2.Data.Job()
+                    {
+                        Configuration = new JobConfiguration()
+                        {
+                            Copy = new JobConfigurationTableCopy()
+                            {
+                                DestinationTable = Destination,
+                                SourceTable = Source,
```

Comment:
I looked into doing that earlier with the ArrayPropertyTransformAttribute, and ran into a whole host of issues, both for code and for usability.  

I don't want to do code using process to aggregate and end to run the operation because then I would have to revamp get to bypass table objects on the pipeline.  This is directly conflicting with one of the frequent uses cases, which is passing a table via pipeline to get an updated table object:
```
$table = $table | Get-BqTable
```

So people would have to manually make an array of tables to pass in, or they would have to do a list request with the label-filter options that I'm going to write after this.  The array option is doable, but messy, and the label option is going to be messy for a similar reason.  they're both going to require a lot of user input and finagling with data types.  

The proposed solution, which I can make note of in the docs, is going to be to use the WriteDisposition -WriteAppend, and just do multiple copy jobs sequentially.  

So instead of saying "copy a,b,c,d to z", it'll be just "copy a to z, copy append b, copy append c, copy append d".  This also works better with the pipeline, because each command takes one argument off the pipeline and copies it at a time.  That's the best compromise that I came up with earlier.  Let me know if you have other ideas for this.